### ISSUE TITLE
The application continues to work in the background after exiting it

### ISSUE THREAD

**Issue by Slawa79** [Arturo254/InnerTune#152]
![IMG_20241214_225308_664.jpg](https://github.com/user-attachments/assets/c911da14-8d49-4a58-b3d6-7801e5eefb83)

![IMG_20241214_225309_251.jpg](https://github.com/user-attachments/assets/1759311b-8764-4c8a-975c-ed8930426853)

![IMG_20241214_225309_392.jpg](https://github.com/user-attachments/assets/a86527c5-3ecf-4ee7-b2d0-589ac93641a8)

The setting is enabled so that the application shuts down after exiting it, but the application continues to run in the background, and the icon in the control panel remains.
  ssmartphone realme C25; android  13

**Comment by jakbin** [comment]
Yes, i am facing similar issue. Even after closing app from recent section, app still run in background. To stop it, i have to force stop it every time from app setting. Please give a button to stop it from background, in place of repeat button.
